core/src/main/java/org/apache/iceberg/util/BinPacking.java

```diff
@@ -62,11 +62,6 @@ public ListPacker(long targetWeight, int lookback, boolean largestBinFirst) {
     private final Function<T, Long> weightFunc;
     private final boolean largestBinFirst;
 
-    public PackingIterable(Iterable<T> iterable, long targetWeight, int lookback,
```

Comment:
Please revert this change. Even if this isn't used, the class is public and we don't need to make unnecessary breaking changes.